<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QDialog" name="StartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="frida.qrc">:/logo/frida-logo-test.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Free Interactive Disassembler</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Version 1.0 Copyright Â© 2017 by Ivo van Poorten</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelBuild">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>build</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameExistingProject">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QPushButton" name="buttonLoadExistingProject">
        <property name="text">
         <string>Load</string>
        </property>
        <property name="icon">
         <iconset resource="frida.qrc">
          <normaloff>:/icons/icons/document-open.png</normaloff>:/icons/icons/document-open.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonBrowseExistingProject">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineExistingProject">
        <property name="text">
         <string>/home/ivo/frida-projects/atmas2.frida</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelExistingProject">
        <property name="text">
         <string>Existing Project</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="space3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <spacer name="space4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameNewProject">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="labelFileDisasm">
        <property name="text">
         <string>File To Disassemble</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineFileDisasm">
        <property name="text">
         <string>/home/ivo/frida-projects/atmas2.xex</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="buttonBrowseFileDisasm">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="buttonNewProject">
        <property name="text">
         <string>New</string>
        </property>
        <property name="icon">
         <iconset resource="frida.qrc">
          <normaloff>:/icons/icons/document-new.png</normaloff>:/icons/icons/document-new.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="space">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelCPUType">
        <property name="text">
         <string>CPU Type</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboFileType"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelFileType">
        <property name="text">
         <string>File Type</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboCPUType"/>
      </item>
      <item row="3" column="2">
       <spacer name="space2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="frida.qrc"/>
 </resources>
 <connections/>
</ui>
